examples/schema.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In python I put all data/schema/dp etc into `data` dir and then use it in `examples`, `tests` etc.